sw/device/tests/sim_dv/flash_escalation_reset_test.c

```diff
@@ -131,24 +202,32 @@ static void alert_handler_config(void) {
 
   dif_alert_handler_escalation_phase_t esc_phases[] = {
       {.phase = kDifAlertHandlerClassStatePhase0,
-       .signal = 0,
+       .signal = 0xFFFFFFFF,  // do not trigger any signal, just wait.
```

Comment:
nit
```suggestion
       .signal = UINT32_MAX,  // do not trigger any signal, just wait.
```

Reply:
fixed.
do we do this for clarity or for other reason? Just curious.